roles/kubernetes-apps/ansible/templates/coredns-deployment-secondary.yml.j2

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
Could we use a single template for this with the `-secondary` appended conditionally by setting a var just for the second template task? That would make this much easier to keep in sync.

Reply:
How would you do so? I can't figure how this should be done to be nice and simple.

Reply:
Will reply inline where the changes will be needed.